media/MetadataViewer/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<!-- 
+Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ 
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+ 
+   http://www.apache.org/licenses/LICENSE-2.0
+ 
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="${index.css}">
+  <script type="text/javascript" src="${index.js}" nonce="${nonce}"></script>
+</head>
+<body>
+  <div id="main-view-item-box" class="main-view-item-box" >
```

Comment:
(For structure) How about creating only a root element and add others from `index.js`

Reply:
ok!!, all div created in index.js like the structure of the relationship viewer,